test/multiverse/suites/concurrent_ruby/Envfile

```diff
@@ -0,0 +1,17 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+# The 1.1.x series of Concurrent Ruby starts to use the error classes
+# we reference in our logic to notice errors after 1.1.5
+# 1.1.4 and below do not use these classes.
+CONCURRENT_RUBY_VERSIONS = [
+  [nil, 2.2],
+  ['1.1.5', 2.2]
+]
+
+gemfile <<-RB
+  gem 'concurrent-ruby', '1.1.5'
```

Comment:
This isn't going to be running it with versions in CONCURRENT_RUBY_VERSIONS, we might want to update this to use the create_gemfile method instead.

Reply:
Thank you for catching this! Resolved in: 2bd96205ac095339b092ed556607f808ae2c49df